integration-tests/utils.py

```diff
@@ -0,0 +1,10 @@
+import random
+from sys import stderr
+from typing import Final
+
+# Neukölln's digest
+PROGRESS_INDICATION_CHARCATERS: Final = "✓→🚴✊★☆⯪𓄁𓅯▶️✴️🪇⚒️🧻🚬🗿🎳⏳🌝☕🐑🐞🌼🪱🌸🏵💮️"
```

Comment:
the i think fairly standard font i use for code doesn't have a lot of those charcaters...